nuxeo-drive-elasticsearch/src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+  <Appenders>
+    <Console name="CONSOLE" target="SYSTEM_OUT">
+      <ThresholdFilter level="WARN" onMatch="ACCEPT" />
+      <PatternLayout pattern="%d{DEFAULT} [%t] %-5p [%C{1}] %m%n" />
+    </Console>
+    <File name="FILE" fileName="target/trace.log" append="false">
+      <ThresholdFilter level="DEBUG" onMatch="ACCEPT" />
+      <PatternLayout pattern="%d{DEFAULT} [%t] %-5p [%C{1}] %m%n" />
+    </File>
+    <File name="REDIS" fileName="target/redis.log" append="false">
+      <PatternLayout pattern="%d{DEFAULT} [%t] %-5p [%C{1}] %m%n" />
+    </File>
+  </Appenders>
+  <Loggers>
+    <Logger name="org.nuxeo" level="info" />
+    <Logger name="org.nuxeo.ecm.core.redis.embedded" level="trace" additivity="false">
+      <AppenderRef ref="REDIS" />
+    </Logger>
+    <Logger name="org.nuxeo.ecm.core.work" level="trace" additivity="false">
+      <AppenderRef ref="REDIS" />
+    </Logger>
```

Comment:
Here if we look at the old implem we had `org.nuxeo.elasticsearch` and `org.nuxeo.drive` in TRACE and no Redis.